core/master_derivable_key.go

```diff
@@ -88,6 +88,6 @@ func (master *MasterDerivableKey) Derive(relativePath string) (*HDKey, error) {
 }
 
 func validateRelativePath(relativePath string) bool {
-	match, _ := regexp.MatchString(`^(\/(\d\d?\d?))+$`, relativePath)
```

Comment:
why is this changed?

Reply:
The primary reason for updating the regular expression in `validateRelativePath` was to facilitate testing of a real block that was signed by a validator whose keys we possess, which allowed us to perform more thorough and realistic testing scenario. Specifically, the issue we faced was related to our inability to derive the validator's key with the relativePath `/3536/0/0`

btw more tests were added due to this change [here](https://github.com/bloxapp/eth2-key-manager/blob/9ccffcf86f2a585f6bc09a90f49b8f6bc4220bb3/core/hd_key_test.go)